front/app/components/IdeasMap/IdeaMapCard.tsx

```diff
@@ -222,13 +222,12 @@ const IdeaMapCard = memo<Props>(
         project.data.attributes.action_descriptor.commenting_idea.enabled;
       const projectHasComments = project.data.attributes.comments_count > 0;
       const showCommentCount = commentingEnabled || projectHasComments;
-      const phaseButNotCurrentPhase = !(
+      const phaseButNotCurrentPhase =
         participationContext?.type === 'phase' &&
         pastPresentOrFuture([
           participationContext.attributes.start_at,
           participationContext.attributes.end_at,
-        ]) === 'present'
-      );
+        ]) !== 'present';
```

Comment:
Do you think it's possible to have "phaseIsCurrent" instead? I find this phaseButNotCurrentPhase just a bit confusing (I know it was present before your changes though) :thinking: 

Maybe:
```
      const phaseIsCurrent = (
        participationContext?.type === 'phase' &&      <-- Do we still need this check if all projects are now timeline?
        pastPresentOrFuture([
          participationContext.attributes.start_at,
          participationContext.attributes.end_at,
        ]) === 'present'
      );
``` 
and then:
```
      const showVoteInput =
        votingMethodConfig && participationContext && phaseIsCurrent;
``` 
or something?

Reply:
Ok sounds good :) :+1: 

I'm still a bit confused by how the `phaseButNotCurrentPhase` is used then - isn't ` !phaseButNotCurrentPhase (line 233)` just checking if it _is_ the current phase (and thus could be replaced by a simple check to see if it's a current phase)? Or am I misunderstanding how this works?

```
  const showVoteInput =
        votingMethodConfig && participationContext && !phaseButNotCurrentPhase;
```

Reply:
Yeah no worries - totally fine to leave it as-is since we're cleaning it up later anyways :) :+1:  Was just a bit confused reading it.